variables.tf

```diff
@@ -156,6 +156,18 @@ variable "subnet_ids" {
   type        = list(string)
 }
 
+variable "subnet_selector_regx" {
+  description = "Select the Subnets from this Regular expression to spin up the Karpenter nodes"
+  type        = string
+  default     = "aft-app-ap-southeast*"
+
+  validation {
+    condition     = can(regex("^aft-app-ap-southeast*|^App*|^Web*", var.subnet_selector_regx))
+    error_message = "Err: invalid subnet selector, must be one of aft-app-ap-southeast*|App*|Web*."
+  }
```

Comment:
make it mandatory instead, since its public module.. better not to use these internal naming convention.

Reply:
my bad.. forgot this is public module :)